tests/functional/deployment/test_acm.py

```
@@ -39,12 +39,13 @@ def wait_for_mcp(cluster):
         # Step 1: Apply IDMS to all clusters
         for cluster in config.clusters:
-            try:
-                apply_idms(cluster)
-            except Exception as e:
-                logger.error(
-                    f"Error applying IDMS on cluster index {cluster.MULTICLUSTER['multicluster_index']}: {e}"
-                )
+            if not cluster.DEPLOYMENT.get("disconnected", False):
+                try:
+                    apply_idms(cluster)
+                except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)